src-tool/Pact/Analyze/Check.hs

```diff
@@ -425,10 +416,9 @@ verifyFunctionInvariants (CheckEnv tables _consts _pDefs moduleData caps gov _de
 
         -- Iterate through each invariant in a single query so we can reuse our
         -- assertion stack.
-        ExceptT $ fmap Right $
-          SBV.query $
-            for2 resultsTable $ \(Located info
-              (AnalysisResult querySucceeds _ prop ksProvs)) -> do
+        res <- ExceptT $ fmap Right $
```

Comment:
```suggestion
        res <- lift $
```

Possible since we're just taking one monad (`SBV IO Foo`) and lifting it into a bigger one ( `ExceptT x (SBV IO Foo)`). But I don't know, maybe `lift` alone is less descriptive and readable than the `ExceptT $ fmap Right` that you have now.